api_v3/services/PartnerService.php

```diff
@@ -76,13 +76,12 @@ public function registerAction( KalturaPartner $partner , $cmsPassword = "" , $t
 			$partner_registration = new myPartnerRegistration ( $parentPartnerId );
 			
 			$ignorePassword = false;
-			if ($existingUser && ($this->getKs()->partner_id == Partner::ADMIN_CONSOLE_PARTNER_ID || $this->getKs()->partner_id == $parentPartnerId)){
```

Comment:
It was true because it was always zero? this part of the condition used for VAR console, then it won't necessarily will be true.